hudi-common/src/test/java/org/apache/hudi/common/table/read/SchemaHandlerTestBase.java

```diff
@@ -67,6 +68,7 @@ public abstract class SchemaHandlerTestBase {
       .map(Schema.Field::name).filter(f -> !f.equals("_hoodie_is_deleted")).toArray(String[]::new));
   protected static final Schema DATA_COLS_ONLY_SCHEMA = generateProjectionSchema("begin_lat", "tip_history", "rider");
   protected static final Schema META_COLS_ONLY_SCHEMA = generateProjectionSchema("_hoodie_commit_seqno", "_hoodie_record_key");
+  protected final HoodieTableMetaClient metaClient = mock(HoodieTableMetaClient.class);
```

Comment:
Will the COW merge handle PR #13699 bring tests to cover the changes in this PR?  I'm thinking if that's the fast way to get test coverage on the changes without (re)writing a lot of tests.